quilt/tools/build.py

```diff
@@ -9,19 +9,17 @@
 
 from six import iteritems
 import yaml
-import pandas as pd
 from tqdm import tqdm
 
 from .store import PackageStore, VALID_NAME_RE, StoreException
 from .const import DEFAULT_BUILDFILE, PACKAGE_DIR_NAME, PARSERS, RESERVED
-from .core import PackageFormat
+from .core import PackageFormat, BuildException, exec_yaml_python
 from .util import FileWithReadProgress
 
-class BuildException(Exception):
-    """
-    Build-time exception class
-    """
-    pass
+# for use by check functions (eval)
```

Comment:
Import them at the place you call `eval`, and pass them in as a locals or globals dict

Reply:
DONE